website/docs/api/spancategorizer.mdx

```diff
@@ -13,6 +13,13 @@ A span categorizer consists of two parts: a [suggester function](#suggesters)
 that proposes candidate spans, which may or may not overlap, and a labeler model
 that predicts zero or more labels for each candidate.
 
+This component comes in two forms: `spancat` and `spancat_singlelabel`. When you
+need to perform multi-label classification on your spans, use `spancat`. The
+`spancat` component uses a `Logistic` layer where the output class probabilities
+are independent for each class. However, if you need to predict at most one true
+class for a span, then use `spancat_singlelabel` <Tag variant="new">3.5.1</Tag>.
```

Comment:
This is a bit tricky, we don't usually use the new tag in running text. Something like this (and then formatted)?

```suggestion
This component comes in two forms: `spancat` and `spancat_singlelabel` (added in spaCy v3.5.1). When you
need to perform multi-label classification on your spans, use `spancat`. The
`spancat` component uses a `Logistic` layer where the output class probabilities
are independent for each class. However, if you need to predict at most one true
class for a span, then use `spancat_singlelabel`.
```